axe.d.ts

```diff
@@ -241,9 +243,35 @@ declare namespace axe {
     helpUrl: string;
     tags: string[];
   }
+  interface SerialDqElement {
+    source: string
+    nodeIndexes: number[]
+    selector: number[],
+    xpath: number[],
+    ancestry: number[],
```

Comment:
These all should be strings, and `selector` and `ancestry` should account for shadow DOM iframes.

```suggestion
    selector: Selector,
    xpath: string[],
    ancestry: Selector,
```